gammapy/modeling/parameter.py

```diff
@@ -220,6 +222,23 @@ def quantity(self, val):
         self.value = val.value
         self.unit = val.unit
 
+    def _check_value_within_bounds(self):
+        """Send a warning if value is outside the min/max range (or at the border within 1%)"""
+        min_nan = np.isnan(self.min)
+        max_nan = np.isnan(self.max)
+        min_ok = self.value > self.min and ~np.isclose(
+            self.value - self.min, 0, atol=1e-2
+        )
+        max_ok = self.value < self.max and ~np.isclose(
+            self.value - self.max, 0, atol=1e-2
+        )
+        if (~min_nan and ~min_ok) or (~max_nan and ~max_ok):
+            warnings.warn(
```

Comment:
I think we should use `log.warn()` here, so that users get access to the logging configuration...